# This test prints return value of various commands. In this particular case
# it's the return value of /bin/false which on Solaris returns 255.
#
# Not suitable for upstream.

--- tests/exec.right	2016-11-28 01:58:36.053039919 -0800
+++ tests/exec.right	2016-11-28 01:58:07.419101147 -0800
@@ -83,7 +83,7 @@ foo 0
 after
 exit code: 1
 exit code: 1
-exit code: 1
+exit code: 255
 exit code: 127
 exit code: 127
 exit code: 127
